src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -32,6 +33,14 @@ private StandardArgumentDefinitions(){}
     public static final String DISABLE_SEQUENCE_DICT_VALIDATION_NAME = "disableSequenceDictionaryValidation";
     public static final String ADD_OUTPUT_SAM_PROGRAM_RECORD = "addOutputSAMProgramRecord";
 
+    public static final String GENOMICSDB_WORKSPACE_LONG_NAME = "GenomicsDBWorkspace";
+    public static final String GENOMICSDB_ARRAY_LONG_NAME = "GenomicsDBArray";
+    public static final String GENOMICSDB_SEGMENT_SIZE_LONG_NAME = "segmentSize";
+    public static final String SIZE_PER_COLUMN_PARTITION_PER_SAMPLE_LONG_NAME = "sizePerColumnPartitionPerSample";
+    public static final String GENOMICSDB_VID_MAP_FILE_LONG_NAME = "GenomicsDBVidMapFile";
+    public static final String GENOMICSDB_CALLSET_MAP_FILE_LONG_NAME = "GenomicsDBCallsetMapFile";
+    public static final String GENOMICSDB_CREATE_WORKSPACE_LONG_NAME = "CreateGenomicsDBWorkspace";
```

Comment:
Can you move these constants to the tool itself, rather than `StandardArgumentDefinitions`, since they are only used in one tool? `StandardArgumentDefinitions` is for arguments that are used in many tools.

Could you also set the short names of the arguments equal to the long names? It's not useful to have cryptic one-letter short names like `S` and `GW` for these arguments.